subprojects/docs/src/samples/userguide/dependencies/declaringBinaryDependenciesWithoutVersion/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'java-library'
```

Comment:
There should be a integration test verifying that the correct version was selected.

Reply:
Uh sorry. I completely forgot that we need to do that.

I added tests for both samples

Reply:
Thanks.